src/ex5/server/clients/pokemon_client.js

```diff
@@ -0,0 +1,34 @@
+const axios = require("axios")
+
+class PokemonClient {
+    constructor() {
+        this.API_URL = 'https://pokeapi.co/api/v2/pokemon/'
+    }
+
+    async getPokemon(id) {
+        try {
+            const response = await axios.get(`${this.API_URL}${id}`)
+            const pokemon = response.data
+
+            return pokemon
+        } catch (error) {
+            console.error(error)
+            throw new Error("Failed to fetch pokemon")
+        }
+    }
+
+    async getManyPokemon(ids) {
```

Comment:
You could use only one function for both cases. Both are fine but your approach is longer and requires more code.